mgnify_backlog/mgnify_handler.py

```diff
@@ -361,8 +361,10 @@ def get_annotation_jobs(self, run_or_assembly_accessions=None, study_accessions=
                 jobs = jobs.distinct()
         if priority:
             jobs = jobs.filter(priority=priority)
-        if status_description:
-            jobs = jobs.filter(status__description=status_description)
+        if status_descriptions:
+            if isinstance(status_descriptions, str):
+                status_descriptions = list([status_descriptions])
```

Comment:
```suggestion
                status_descriptions = [status_descriptions]
```
Already making a list, no need for type casting